### PR TITLE
Tick: Refactored render() to three separate functions:

### PR DESCRIPTION
**Pull request by larsac07**
- Allows for reuse, especially in GridAxis/CellView and TreeGrid, where we'd have to redo the entire rendering (code duplication, or modify before or after rendering (unnecessarily heavy, GridAxis/CellView does this now) 
- Easier to maintain

### CODE REVIEW

**Comment by larsac07** [comment]
FYI, all tests running green

**Comment by jon-a-nygaard** [comment]
I am much in favour of splitting up the Tick.render logic, because of simple reason to have seperation of concerns. For instance this makes it easier to focus on the label logic withouth beeing afraid of breaking either the mark or gridLine logic. IMHO.

**Comment by oysteinmoseng** [comment]
I think everything is better with smaller isolated functions. Easier to maintain & expand & wrap. If all the tests pass, performance is the same, and the filesize is not much bigger I don't see any reason not to go for this.

File: js/parts/Tick.js [hunk 1]
```diff
@@ -259,51 +259,31 @@ H.Tick.prototype = {
 	},
 
 	/**
-	 * Put everything in place
-	 *
-	 * @param index {Number}
-	 * @param old {Boolean} Use old coordinates to prepare an animation into new position
+	 * [renderGridLine description]
```
Comment (TorsteinHonsi) [inline js/parts/Tick.js]: Whoops

File: js/parts/Tick.js [hunk 2]
```diff
@@ -320,23 +300,59 @@ H.Tick.prototype = {
...
 				gridLine[tick.isNew ? 'attr' : 'animate']({
 					d: gridLinePath,
 					opacity: opacity
 				});
 			}
 		}
+	},
+
+	/**
+	 * [renderMark description]
```
Comment (TorsteinHonsi) [inline js/parts/Tick.js]: Whoopsy

File: js/parts/Tick.js (hunk 1 above)
Comment (larsac07) [inline js/parts/Tick.js]: I made a whoopsie

File: js/parts/Tick.js (hunk 2 above)
Comment (larsac07) [inline js/parts/Tick.js]: I made another whoopsie

**Comment by larsac07** [comment]
@TorsteinHonsi just fixed the whoopsies

**Pull request merged**